binary_search_tree/tree.py

```diff
@@ -1,3 +1,8 @@
+from logging import root
+from multiprocessing.sharedctypes import Value
+from unittest import result
```

Comment:
It looks like none of these `import` statements got used. My guess is that they were added by your IDE when something was autocompleted, but it's good practice to remove them if you don't end up using them to just keep your code clean.